diff --git a/conf/rules.py b/conf/rules.py
index a938737..bb1c179 100644
--- a/conf/rules.py
+++ b/conf/rules.py
@@ -23,11 +23,11 @@ dispatch_rules = [
     'header_name': 'Event-Name',
     'header_value': 'HEARTBEAT',
     'persistent': True,
-    'target': 'switchlets.heartbeat.example.Monitor'
+    'target': 'freepy.switchlets.heartbeat.example.Monitor'
   }
   #,{
   #  'header_name': 'FreeSWITCH-Hostname',
   #  'header_pattern': 'Freeswitch.*',
   #  'target': 'other.application.actor'
   #}
-]
\ No newline at end of file
+]
diff --git a/conf/settings.py b/conf/settings.py
index 0e370a1..03ee22d 100644
--- a/conf/settings.py
+++ b/conf/settings.py
@@ -46,7 +46,7 @@ dispatcher_services = [
       'ReceiveTimeoutCommand',
       'StopTimeoutCommand'
     ],
-    'target': 'lib.services.TimerService'  # The path to the service.
+    'target': 'freepy.lib.services.TimerService'  # The path to the service.
   }
 ]
 
diff --git a/lib/server.py b/lib/server.py
index ac94dcd..b97d3bc 100644
--- a/lib/server.py
+++ b/lib/server.py
@@ -17,12 +17,12 @@
 #
 # Thomas Quintana <quintana.thomas@gmail.com>
 
-from conf.settings import *
-from lib.commands import *
-from lib.core import *
-from lib.esl import *
-from lib.fsm import *
-from lib.services import *
+from freepy.conf.settings import *
+from freepy.lib.commands import *
+from freepy.lib.core import *
+from freepy.lib.esl import *
+from freepy.lib.fsm import *
+from freepy.lib.services import *
 from pykka import ActorRegistry, ThreadingActor
 from twisted.internet import reactor
 
diff --git a/run.py b/run.py
index 3460f7c..d8afdbe 100644
--- a/run.py
+++ b/run.py
@@ -17,7 +17,7 @@
 #
 # Thomas Quintana <quintana.thomas@gmail.com>
 
-from lib.server import FreepyServer
+from freepy.lib.server import FreepyServer
 
 def main():
   server = FreepyServer()
diff --git a/switchlets/heartbeat/example.py b/switchlets/heartbeat/example.py
index 2ade8fa..bc83c95 100644
--- a/switchlets/heartbeat/example.py
+++ b/switchlets/heartbeat/example.py
@@ -19,10 +19,10 @@
 #
 # Thomas Quintana <quintana.thomas@gmail.com>
 
-from lib.commands import *
-from lib.core import *
-from lib.fsm import *
-from lib.server import Event, RegisterJobObserverCommand, UnregisterJobObserverCommand
+from freepy.lib.commands import *
+from freepy.lib.core import *
+from freepy.lib.fsm import *
+from freepy.lib.server import Event, RegisterJobObserverCommand, UnregisterJobObserverCommand
 
 import logging
 import urllib
diff --git a/switchlets/ivr/menu.py b/switchlets/ivr/menu.py
index 18742e0..783db9a 100644
--- a/switchlets/ivr/menu.py
+++ b/switchlets/ivr/menu.py
@@ -19,10 +19,10 @@
 #
 # Thomas Quintana <quintana.thomas@gmail.com>
 
-from lib.commands import *
-from lib.core import *
-from lib.fsm import *
-from lib.server import Event, RegisterJobObserverCommand, UnregisterJobObserverCommand
+from freepy.lib.commands import *
+from freepy.lib.core import *
+from freepy.lib.fsm import *
+from freepy.lib.server import Event, RegisterJobObserverCommand, UnregisterJobObserverCommand
 
 import logging
 import urllib
diff --git a/switchlets/ivr/test.py b/switchlets/ivr/test.py
index 8c8e77d..4f5581d 100644
--- a/switchlets/ivr/test.py
+++ b/switchlets/ivr/test.py
@@ -19,7 +19,7 @@
 #
 # Thomas Quintana <quintana.thomas@gmail.com>
 
-from switchlets.ivr.menu import *
+from freepy.switchlets.ivr.menu import *
 from unittest import TestCase
 
 class TestIVRMenu(AbstractIVRMenu):
diff --git a/tests/command_tests.py b/tests/command_tests.py
index a52e33b..4d6104b 100644
--- a/tests/command_tests.py
+++ b/tests/command_tests.py
@@ -19,7 +19,7 @@
 #
 # Cristian Groza <frontc18@gmail.com>
 
-from lib.commands import *
+from freepy.lib.commands import *
 from unittest import TestCase, expectedFailure
 
 class BackgroundCommandTests(TestCase):
diff --git a/tests/fsm_tests.py b/tests/fsm_tests.py
index 87968a8..4380936 100644
--- a/tests/fsm_tests.py
+++ b/tests/fsm_tests.py
@@ -17,7 +17,7 @@
 #
 # Thomas Quintana <quintana.thomas@gmail.com>
 
-from lib.fsm import *
+from freepy.lib.fsm import *
 from unittest import TestCase, expectedFailure
 
 class AbstractLightBulb(FiniteStateMachine):
diff --git a/tests/server_tests.py b/tests/server_tests.py
index 1d838ef..60db330 100644
--- a/tests/server_tests.py
+++ b/tests/server_tests.py
@@ -17,8 +17,8 @@
 #
 # Thomas Quintana <quintana.thomas@gmail.com>
 
-from lib.core import *
-from lib.server import *
+from freepy.lib.core import *
+from freepy.lib.server import *
 from pykka import ActorRegistry, ThreadingActor
 from unittest import TestCase
 
diff --git a/tests/services_tests.py b/tests/services_tests.py
index 6fbb7b8..74fe9b8 100644
--- a/tests/services_tests.py
+++ b/tests/services_tests.py
@@ -17,8 +17,8 @@
 #
 # Thomas Quintana <quintana.thomas@gmail.com>
 
-from lib.core import *
-from lib.services import *
+from freepy.lib.core import *
+from freepy.lib.services import *
 from pykka import ActorRegistry
 from unittest import TestCase
 
